examples/webgl_materials_video.html

```diff
@@ -244,7 +244,7 @@
 				camera.updateProjectionMatrix();
 
 				renderer.setSize( window.innerWidth, window.innerHeight );
-				composer.reset();
+				composer.setSize( window.innerWidth, window.innerHeight );
```

Comment:
As is was before.

`composer.reset()` should auto-size the composer based on the renderer's drawing buffer size.